packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -48,6 +49,31 @@ export const MONGODB_VERSION = (process.env.MONGODB_VERSION || '5.0.6')
   // we can't just map it to 5.0.0
   .replace(/x/g, '999');
 
+export const MONGODB_USE_ENTERPRISE =
+  process.env.MONGODB_USE_ENTERPRISE ?? 'no';
+
+// This will just match any latest available release. We will need to update it
+// when next major version will start rolling out (this is not just something
+// like 999.999.999 because we are checking for `< 7.0.0-alpha` in one test)
+const LATEST_ALPHA = '6.999.999';
+
+export const serverSatisfies = (
+  semverCondition: string,
+  enterpriseExact?: boolean
+) => {
+  return (
+    semver.satisfies(
+      MONGODB_VERSION === 'latest-alpha' ? LATEST_ALPHA : MONGODB_VERSION,
```

Comment:
So this is a bit awkward while we are relying on env and not actual running version, but now that this all is in one place, we might have easier job if we decide to change this in the future?